services/order.py

```diff
@@ -0,0 +1,28 @@
+from db.models import MovieSession, Order, Ticket
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: str | None = None
+) -> None:
+    user = get_user_model().objects.get(username=username)

```

Comment:
The `get` method does not handle the case where a user with the given username does not exist. Consider using `get_object_or_404` or handling the `DoesNotExist` exception to provide a more user-friendly error message.